cmds/core/mv/mv.go

```diff
@@ -31,16 +31,16 @@ var (
 	noClobber = flag.Bool("n", false, "do not overwrite an existing file")
 )
 
-func moveFile(source string, dest string) error {
-	if *noClobber {
+func moveFile(source string, dest string, update bool, noClobber bool) error {
```

Comment:
one last thing: could it be
moveFile(dest, source string, update, noClobber bool)  error

Reply:
It can, but it will require to swap `files[0], files[1] = files[1], files[0]`, because move command with two operands expects that first operand is a source. So I would not go this far.